File: 김찬우/프로그래머스 /Lv.2/멀리뛰기.java
class Solution {
    public long solution(int n) {
        
        int[] result = new int[2001];
        result[1] = 1;
        result[2] = 2;
        
        for(int i=3; i<=n;i++){
            
            result[i] = (result[i-1]+result[i-2])%1234567;
            
        }
        
        
        return result[n];
    }
}

File: 조준장/level2/멀리 뛰기.kt
class Solution {
    fun solution(n: Int): Long {
        var answer: Long = 0
        var dp : IntArray = IntArray(n+1){1}
        (2..n).forEach{dp[it] = (dp[it-1] + dp[it-2]) % 1234567}
        return dp[n] % 1234567
    }
}
